src/components/BlockSpreadsheets/BlocksTable.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```

Comment:
Is it always necessary to import `React` although it's never used?

Reply:
React is necessary because I'm using jsx